integreat_cms/cms/utils/pdf_utils.py

```diff
@@ -18,6 +18,8 @@
 
 from xhtml2pdf import pisa
 
+
+from ..utils.text_utils import truncate_bytewise
```

Comment:
```suggestion
from .text_utils import truncate_bytewise
```
Just a minor detail - since you're already in the utils package, you don't need to traverse to the parent package to import from the text utils.